packages/tools/src/utilities/dynamicVolume/getDataInTime.ts

```diff
@@ -0,0 +1,104 @@
+import { utilities, cache } from '@cornerstonejs/core';
+
+/**
+ *
+ * @param dynamicVolume: 4D volume to compute time point data from
+ * @param options: frameNumbers: which frames to us as timepoints, if left
+ * blank, gets data timepoints over all frames
+ * maskVolumeId: segmentationId to get timepoint data of
+ * imageCoordinate: image coordinate to get timepoint data of
+ * @returns
+ */
+function getDataInTime(
+  // dynamicVolumeId: string,
+  dynamicVolume: any,
+  options: {
+    frameNumbers?;
+    maskVolumeId?;
+    imageCoordinate?;
+  }
+): number[] | number[][] {
+  let frames;
+  let dataInTime;
+
+  // if frameNumbers is not provided, all frames are selected
+  if (!options.frameNumbers) {
+    frames = [...Array(dynamicVolume.numTimePoints).keys()];
+  } else {
+    frames = options.frameNumbers;
+  }
+
+  // Throws error if neither maskVolumeId or imageCoordinate is given,
+  // throws error if BOTH maskVolumeId and imageCoordinate is given
+  if (!options.maskVolumeId && !options.imageCoordinate) {
+    throw new Error('No ROI provided');
```

Comment:
more context, 'You only need to provide one ....'

Reply:
Added comment before error explanations "You only need to provide either maskVolumeId OR imageCoordinate."